daemons/controld/controld_remote_ra.c

```diff
@@ -1451,15 +1451,21 @@ remote_ra_process_maintenance_nodes(xmlNode *xml)
         const char *id = pcmk__xe_id(node);
 
         if (id == NULL) {
+            CRM_LOG_ASSERT(id != NULL);
```

Comment:
Is this really necessary?

Reply:
Logging is never really necessary. I figured we'd want to know if `id` was `NULL` somehow. Normally I'd use `CRM_CHECK(id != NULL, <action>)`, but we can't use that with `continue`.